pression/data/Compressor.h

```diff
@@ -0,0 +1,126 @@
+
+/* Copyright (c) 2016, Stefan.Eilemann@epfl.ch
+ *
+ * This file is part of Pression <https://github.com/Eyescale/Pression>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+#include <pression/data/api.h>
+#include <pression/data/types.h>
+#include <lunchbox/buffer.h> // used inline
+#include <lunchbox/compiler.h> // LB_UNUSED
+#include <lunchbox/debug.h> // LBUNIMPLEMENTED
+
+namespace pression
+{
+namespace data
```

Comment:
I haven't found any header file inside the data directory with documentation for this namespace.